src/style.css

```diff
@@ -1 +1,115 @@
-/* add styles here */
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('/src/fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('/src/fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: 700;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  color: rgba(61, 78, 97, 1);
+  background-color: #fff;
+}
+
+.search-form {
+  display: flex;
+  justify-content: left;
+  margin: 20px auto;
+  padding-left: 48px;
+  width: 100%;
+}
+
+.search-form .big {
+  max-width: 900px;
+}
+
+.search-form .small {
+  max-width: 427px;
+}
+
+.search-form + .search-form {
+  margin-top: 20px;
+}
+
+.search-bar {
+  width: 100%;
+  padding: 50px;
+  padding: 10px 40px;
```

Comment:
There is a redundancy in the padding property for the `.search-bar` class. The padding is defined twice: `padding: 50px;` and `padding: 10px 40px;`. The second declaration will override the first one. Consider removing the redundant padding declaration.